ophyd/runengine/runengine.py

```diff
@@ -275,7 +275,7 @@ def _demunge_names(self, names):
         msg = ''.join('{}\t'.format(name) for name in unique_names)
         return msg
 
-    def _get_data_keys(self, positioners=None, detectors=None):
+    def _get_data_keys(self, positioners=None, detectors=None, **kwargs):
```

Comment:
Perhaps the HXN configuration is incorrect, but things like `settle_time` were getting passed along and causing that to break:

```python
TypeError: _get_data_keys() got an unexpected keyword argument 'settle_time'
```

A better solution, I suppose, would be to just give `_get_data_keys` only the arguments it expects.